2024-04-06 20:33:44,235 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-06 20:33:44,235 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Configure stats pid to 28589
2024-04-06 20:33:44,235 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Loading settings from /Users/artemon/.config/wandb/settings
2024-04-06 20:33:44,235 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Loading settings from /Users/artemon/Library/Mobile Documents/com~apple~CloudDocs/Programming/python_projects/RNN/wandb/settings
2024-04-06 20:33:44,235 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-06 20:33:44,235 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-04-06 20:33:44,235 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-06 20:33:44,235 INFO    MainThread:28589 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-06 20:33:44,236 INFO    MainThread:28589 [wandb_init.py:_log_setup():521] Logging user logs to /Users/artemon/Library/Mobile Documents/com~apple~CloudDocs/Programming/python_projects/RNN/wandb/run-20240406_203344-t760ix2m/logs/debug.log
2024-04-06 20:33:44,236 INFO    MainThread:28589 [wandb_init.py:_log_setup():522] Logging internal logs to /Users/artemon/Library/Mobile Documents/com~apple~CloudDocs/Programming/python_projects/RNN/wandb/run-20240406_203344-t760ix2m/logs/debug-internal.log
2024-04-06 20:33:44,236 INFO    MainThread:28589 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x324296b90>
2024-04-06 20:33:44,236 INFO    MainThread:28589 [wandb_init.py:init():561] calling init triggers
2024-04-06 20:33:44,236 INFO    MainThread:28589 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-06 20:33:44,236 INFO    MainThread:28589 [wandb_init.py:init():611] starting backend
2024-04-06 20:33:44,236 INFO    MainThread:28589 [wandb_init.py:init():615] setting up manager
2024-04-06 20:33:44,236 INFO    MainThread:28589 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-04-06 20:33:44,238 INFO    MainThread:28589 [wandb_init.py:init():623] backend started and connected
2024-04-06 20:33:44,241 INFO    MainThread:28589 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-06 20:33:44,241 INFO    MainThread:28589 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-06 20:33:44,241 INFO    MainThread:28589 [wandb_init.py:init():715] updated telemetry
2024-04-06 20:33:44,257 INFO    MainThread:28589 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-06 20:33:44,788 INFO    MainThread:28589 [wandb_run.py:_on_init():2357] communicating current version
2024-04-06 20:33:44,985 INFO    MainThread:28589 [wandb_run.py:_on_init():2366] got version response 
2024-04-06 20:33:44,986 INFO    MainThread:28589 [wandb_init.py:init():799] starting run threads in backend
2024-04-06 20:33:45,054 INFO    MainThread:28589 [wandb_run.py:_console_start():2335] atexit reg
2024-04-06 20:33:45,054 INFO    MainThread:28589 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-06 20:33:45,054 INFO    MainThread:28589 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-06 20:33:45,054 INFO    MainThread:28589 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-06 20:33:45,054 INFO    MainThread:28589 [wandb_init.py:init():842] run started, returning control to user process
2024-04-06 20:33:51,016 INFO    MainThread:28589 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-06 20:33:51,016 INFO    MainThread:28589 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-06 20:37:22,783 INFO    MainThread:28589 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-06 20:37:22,786 INFO    MainThread:28589 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-06 20:37:22,786 INFO    MainThread:28589 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-06 20:37:23,187 INFO    MainThread:28589 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-06 20:37:23,191 INFO    MainThread:28589 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-06 20:37:23,191 INFO    MainThread:28589 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-06 20:37:24,892 INFO    MainThread:28589 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-06 20:37:24,898 INFO    MainThread:28589 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-06 20:37:24,898 INFO    MainThread:28589 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-06 20:37:53,999 INFO    MainThread:28589 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-06 20:37:54,001 INFO    MainThread:28589 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-06 20:37:54,001 INFO    MainThread:28589 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-06 20:38:02,645 INFO    MainThread:28589 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-06 20:38:02,670 INFO    MainThread:28589 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-06 20:38:02,670 INFO    MainThread:28589 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-06 20:38:03,419 INFO    MainThread:28589 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-06 20:38:03,420 INFO    MainThread:28589 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-06 20:38:03,420 INFO    MainThread:28589 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-06 20:38:04,190 INFO    MainThread:28589 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-06 20:38:04,191 INFO    MainThread:28589 [wandb_run.py:_finish():2064] finishing run artem-tm/PRAC_RNN/t760ix2m
2024-04-06 20:38:04,191 ERROR   MainThread:28589 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-06 20:38:04,192 INFO    MainThread:28589 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-06 20:38:04,192 INFO    MainThread:28589 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-06 20:38:04,192 INFO    MainThread:28589 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-06 20:38:04,192 INFO    MainThread:28589 [wandb_run.py:_restore():2287] restore
2024-04-06 20:38:04,192 INFO    MainThread:28589 [wandb_run.py:_restore():2293] restore done
2024-04-06 20:38:10,115 INFO    MainThread:28589 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-06 20:38:10,115 INFO    MainThread:28589 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-06 20:38:10,118 INFO    MainThread:28589 [wandb_run.py:_footer_sync_info():3895] logging synced files
